scripts/01_Seurat_QC.Rmd

```diff
@@ -1,115 +1,110 @@
 ---
 title: "Seurat QC and Object for `r params$sample_name`"
-author: 'User ID: `r Sys.getenv("USER")`'
-date: "`r Sys.Date()`"
+author: 'Antonia Chroni <chronia@chop.edu> for D3B'
+date: "2023"
 output:
-  html_document:
-    fig_height: 7
-    fig_width: 9
-    keep_md: yes
-    toc: yes
-    df_print: paged
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
 params:
-  out_path: './' # path to dir to save all output files
-  data_path: '/path/to/10x/output/dir/filtered_feature_bc_matrix/' # path to the directory with the filtered_feature_bc_matrix
```

Comment:
I think we need to keep these parameters around so they can be altered when running this as an Rscript